bagit.py

```diff
@@ -1361,7 +1380,8 @@ def main():
             try:
                 bag = Bag(bag_dir)
                 # validate throws a BagError or BagValidationError
-                bag.validate(processes=args.processes, fast=args.fast)
+                bag.validate(processes=args.processes, fast=args.fast,
+                    completeness=args.completeness)
```

Comment:
I would prefer the hanging indent align the argument with the opening parentheses - basically the first example in https://www.python.org/dev/peps/pep-0008/#indentation